gammapy/datasets/spectrum.py

```diff
@@ -28,18 +28,18 @@ def plot_fit(
         Parameters
         ----------
         ax_spectrum : `~matplotlib.axes.Axes`
-            Axes to plot spectrum on
+            Axes to plot spectrum on. Default is None.
         ax_residuals : `~matplotlib.axes.Axes`
-            Axes to plot residuals on
+            Axes to plot residuals on. Default is None.
         kwargs_spectrum : dict
```

Comment:
I guess `kwargs` are always optional.

```suggestion
        kwargs_spectrum : dict, optional
```

There are a couple of these cases.